### ISSUE TITLE
User API fails to start

### ISSUE THREAD

**Issue by fridex** [thoth-station/user-api#1708]
**Describe the bug**

```
[2022-03-09 12:17:17 +0100] [2783728] [INFO] Starting gunicorn 20.1.0
[2022-03-09 12:17:17 +0100] [2783728] [INFO] Listening at: http://127.0.0.1:8000 (2783728)
[2022-03-09 12:17:17 +0100] [2783728] [INFO] Using worker: sync
[2022-03-09 12:17:17 +0100] [2783770] [INFO] Booting worker with pid: 2783770
[2022-03-09 12:17:17 +0100] [2783770] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/home/fpokorny/.local/share/virtualenvs/user-api-RH9-ePKu/lib/python3.8/site-packages/gunicorn/arbiter.py", line 589, in spawn_worker
    worker.init_process()
  File "/home/fpokorny/.local/share/virtualenvs/user-api-RH9-ePKu/lib/python3.8/site-packages/gunicorn/workers/base.py", line 134, in init_process
    self.load_wsgi()
  File "/home/fpokorny/.local/share/virtualenvs/user-api-RH9-ePKu/lib/python3.8/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/home/fpokorny/.local/share/virtualenvs/user-api-RH9-ePKu/lib/python3.8/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/home/fpokorny/.local/share/virtualenvs/user-api-RH9-ePKu/lib/python3.8/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
    return self.load_wsgiapp()
  File "/home/fpokorny/.local/share/virtualenvs/user-api-RH9-ePKu/lib/python3.8/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/home/fpokorny/.local/share/virtualenvs/user-api-RH9-ePKu/lib/python3.8/site-packages/gunicorn/util.py", line 359, in import_app
    mod = importlib.import_module(module)
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/fpokorny/git/thoth-station/user-api/thoth/user_api/openapi_server.py", line 28, in <module>
    import connexion
  File "/home/fpokorny/.local/share/virtualenvs/user-api-RH9-ePKu/lib/python3.8/site-packages/connexion/__init__.py", line 5, in <module>
    from .apis import AbstractAPI  # NOQA
  File "/home/fpokorny/.local/share/virtualenvs/user-api-RH9-ePKu/lib/python3.8/site-packages/connexion/apis/__init__.py", line 1, in <module>
    from .abstract import AbstractAPI  # NOQA
  File "/home/fpokorny/.local/share/virtualenvs/user-api-RH9-ePKu/lib/python3.8/site-packages/connexion/apis/abstract.py", line 10, in <module>
    from ..operations import make_operation
  File "/home/fpokorny/.local/share/virtualenvs/user-api-RH9-ePKu/lib/python3.8/site-packages/connexion/operations/__init__.py", line 1, in <module>
    from .abstract import AbstractOperation  # noqa
  File "/home/fpokorny/.local/share/virtualenvs/user-api-RH9-ePKu/lib/python3.8/site-packages/connexion/operations/abstract.py", line 10, in <module>
    from ..decorators.response import ResponseValidator
  File "/home/fpokorny/.local/share/virtualenvs/user-api-RH9-ePKu/lib/python3.8/site-packages/connexion/decorators/response.py", line 12, in <module>
    from .validation import ResponseBodyValidator
  File "/home/fpokorny/.local/share/virtualenvs/user-api-RH9-ePKu/lib/python3.8/site-packages/connexion/decorators/validation.py", line 9, in <module>
    from werkzeug import FileStorage
ImportError: cannot import name 'FileStorage' from 'werkzeug' (/home/fpokorny/.local/share/virtualenvs/user-api-RH9-ePKu/lib/python3.8/site-packages/werkzeug/__init__.py)
```

**Comment by mayaCostantini** [comment]
The latest version of `connexion` which still has this import present is `2.3.0`. Let's try to use this version and see if this solves the issue.

**Comment by fridex** [comment]
Related: https://github.com/spec-first/connexion/pull/1044

**Comment by fridex** [comment]
We have pinned werzeug==1.0.1 based on https://github.com/thoth-station/user-api/pull/1421 it looks like we could create prescriptions for these cases.

**Comment by mayaCostantini** [comment]
Sure @fridex I will create them 👍🏻

**Comment by harshad16** [comment]
During pair programming to fix this issue, @mayaCostantini  and I found this. 
This issues: `ImportError: cannot import name 'FileStorage' from 'werkzeug' `
Is due to the reason, that **FileStorage'** is moved to `werkzeug.datastructures` in `werkzeug` version 2.x above, and it is already been satisfied by the version  of `connexion`=`2.12.0`  which is the latest.

we don't have to downgrade the connexion.

However, the flask version change causes this issue:
https://github.com/thoth-station/user-api/issues/1701
it was addressed that **jinjautils** requires to be above 3.x

which is satisfied with flask 2.x .

With the change of flask upgrade we fall back to issue: https://github.com/thoth-station/management-api/issues/762
At that moment, we had used the hotfix of pinning flask to 1.1.4.
However it is stated in this issue: https://github.com/pallets/flask/issues/4159
that `flask_script` is deprecated.
we should try to migrate from flask_script to flask cli.
this would solve all the issues.

In Nutshell:
- Make `flask = >1.0.0`  to fix: https://github.com/thoth-station/user-api/issues/1701
- Migrate to flask cli from flask_script to fix: https://github.com/thoth-station/management-api/issues/762

References:
https://blog.miguelgrinberg.com/post/migrating-from-flask-script-to-the-new-flask-cli

**Comment by fridex** [comment]
If I read this correctly, should we create prescriptions for these cases?

**Comment by mayaCostantini** [comment]
@fridex Definitely 👍🏻 We are just making sure this is the right reason behind user-api issues and that everything is working again before, and I will create those prescriptions then.

**Comment by fridex** [comment]
Perfect, thanks! 👍🏻

**Comment by fridex** [comment]
It looks like the issue has been fixed with #1718. Thanks @mayaCostantini @harshad16 for making this happen 👍🏻 

/close
